### ISSUE TITLE
Color contrast violations

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/dpul#1265]
### Error in spec/features/accessibility_spec.rb

1) accessibility when visiting catalog show page complies with WCAG

       color-contrast: Elements must have sufficient color contrast (serious)
       https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI
       The following 2 nodes violate this rule:
           
               Selector: .nav-link
               HTML: <a class="nav-link" href="/exhibit-title-1">Home</a>
               Fix any of the following:
               - Element has insufficient color contrast of 2.5 (foreground color: #757575, background color: #bfbfbf, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .active > .truncated-value
               HTML: <span class="truncated-value">test item</span>
               Fix any of the following:
               - Element has insufficient color contrast of 4.3 (foreground color: #6c757d, background color: #f5f5f5, font size: 10.8pt (14.4px), font weight: normal). Expected contrast ratio of 4.5:1

2) accessibility when visiting catalog page with search exhibit catalog path complies with WCAG

       color-contrast: Elements must have sufficient color contrast (serious)
       https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI
       The following 2 nodes violate this rule:
           
               Selector: .nav-link
               HTML: <a class="nav-link" href="/slug-4">Home</a>
               Fix any of the following:
               - Element has insufficient color contrast of 2.5 (foreground color: #757575, background color: #bfbfbf, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .breadcrumb-item.active > .truncated-value
               HTML: <span class="truncated-value">Search results</span>
               Fix any of the following:
               - Element has insufficient color contrast of 4.3 (foreground color: #6c757d, background color: #f5f5f5, font size: 10.8pt (14.4px), font weight: normal). Expected contrast ratio of 4.5:1

3) accessibility when visiting edit page complies with WCAG

       color-contrast: Elements must have sufficient color contrast (serious)
       https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI
       The following 1 node violate this rule:
           
               Selector: .nav-link
               HTML: <a class="nav-link" href="/exhibit-title-1">Home</a>
               Fix any of the following:
               - Element has insufficient color contrast of 2.5 (foreground color: #757575, background color: #bfbfbf, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

4) accessibility when visiting language create edit page complies with WCAG

       color-contrast: Elements must have sufficient color contrast (serious)
       https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI
       The following 3 nodes violate this rule:
           
               Selector: .nav-link[href$="slug-7"]
               HTML: <a class="nav-link" href="/slug-7">Home</a>
               Fix any of the following:
               - Element has insufficient color contrast of 2.5 (foreground color: #757575, background color: #bfbfbf, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .breadcrumb-item.active > .truncated-value
               HTML: <span class="truncated-value">General</span>
               Fix any of the following:
               - Element has insufficient color contrast of 4.3 (foreground color: #6c757d, background color: #f5f5f5, font size: 10.8pt (14.4px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: #another-email
               HTML: <button id="another-email" class="btn btn-sm btn-info">Add new recipient</button>
               Fix any of the following:
               - Element has insufficient color contrast of 3.04 (foreground color: #ffffff, background color: #17a2b8, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

**Comment by hackartisan** [comment]
This was a duplicate of #1217
